src/uwtools/config/formats/base.py

```diff
@@ -50,6 +46,43 @@ def __repr__(self) -> str:
 
     # Private methods
 
+    def _characterize_values(self, values: dict, parent: str) -> tuple[list, list]:
```

Comment:
Here and below, several formerly-public methods now marked private as they're likely to mystify end users if they appear in docs.